src/inc/quic_versions.h

```diff
@@ -14,28 +14,28 @@
 //
 // The QUIC version numbers, in network byte order.
 //
-#define QUIC_VERSION_VER_NEG    0x00000000      // Version for 'Version Negotiation'
-#define QUIC_VERSION_1          0x01000000      // First official version
-#define QUIC_VERSION_MS_1       0x0000cdab      // First Microsoft version (currently same as latest draft)
-#define QUIC_VERSION_DRAFT_27   0x1b0000ff      // IETF draft 27
-#define QUIC_VERSION_DRAFT_28   0x1c0000ff      // IETF draft 28
-#define QUIC_VERSION_DRAFT_29   0x1d0000ff      // IETF draft 29
+#define QUIC_VERSION_VER_NEG    0x00000000U      // Version for 'Version Negotiation'
```

Comment:
nit: can you fix the comments back to where they were?

Reply:
You mean the indentation?

Reply:
Yes. Just delete a space before it please.